src/rdb.c

```diff
@@ -166,8 +166,8 @@ int rdbSaveLen(rio *rdb, uint64_t len) {
     size_t nwritten;
 
     if (len < (1<<6)) {
-        /* Save a 6 bit len */
-        buf[0] = (len&0xFF)|(RDB_6BITLEN<<6);
+        /* Save a 6 bit len, Currently RDB_6BITLEN is 0 ,no need to operate buf[0] with `|(RDB_6BITLEN<<6)`*/
+        buf[0] = len&0xFF;
```

Comment:
`(RDB_6BITLEN<<6)` is a constant expression so if it is 0 it will be optimized away. It's better not to assume anything about the value of RDB_6BITLEN.